lib/member_row.rb

```diff
@@ -32,6 +36,14 @@ class MemberRow < Scraped::HTML
 
   private
 
+  def prefixes
+    %w(Prof.)
+  end
+
+  def name_parts
+    noko[1].text.strip.sub(/^Hon.? /, '').split(' ')
```

Comment:
Let's be consistent how we did this in Malta, and have an "unwanted" list with 'Hon'. The more we can do this the same way across multiple scrapers, the easier it will be to discover what the _real_ edge-cases are. Doing things differently in each scraper makes it much harder to extract the commonalities out to a library.